### DISCUSSION TITLE
How do I listen for routing changes in the WebView？

### DISCUSSION THREAD

**Discussion by GuoJikun** [tauri-apps/tauri#5036] [Q&A]
How do I listen for routing changes in the WebView？

**Comment by GuoJikun** [comment]
how watching of location.href change?

**Reply by abisar** [reply]
How are you handling routing? Are you using any frameworks (react, vue, ...)? Or does the page change with a classic anchor tag that does a full load?

**Reply by GuoJikun** [reply]
The page in the WebView is provided by a third party, and my purpose is to get the hash part of a URL in the WebView.

**Reply by zhanglun** [reply]
you can access hash  with `window.location.hash`
